api/src/main/java/org/openmrs/module/pihcore/identifier/haiti/ConfigureHaitiIdGenerators.java

```diff
@@ -79,7 +77,7 @@ else if (config.getDossierIdentifierPrefix() != null) {
             SequentialIdentifierGenerator sequentialIdentifierGenerator = configureHaitiIdGenerators
                     .sequentialIdentifierGeneratorForDossier(dossierIdentifierType,
                             config.getDossierIdentifierPrefix().toString(),
-                            ZL_DOSSIER_NUMBER_IDENTIFIER_SOURCE_UUID);
+                            "9dd9bdf3-4b57-47c3-b731-1000dbdef5d8");
```

Comment:
Yeah, having constants in PihCoreConstants gives the false impression that they are used in multiple places in the code. This makes it harder to refactor, since that constant is technically part of the public API of the module, and you need to search across any possible client repositories for usages. It mystifies the usage of that string or variable.

I've put it at the top of the class as a private member, though I don't think it adds anything to the readability of the code, since modern IDEs show function parameter names inline.